react-accordion-component/src/accordion.jsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+class Accordion extends React.Component {
+  constructor(props) {
+    super(props);
+    this.topics = props.topics;
+
+    this.state = {
+      selectedTopicId: 0
+    };
+
+    this.handleClick = this.handleClick.bind(this);
+  }
+
+  handleClick(event) {
+    if (Number(event.target.id) === Number(this.state.selectedTopicId)) {
+      this.setState({ selectedTopicId: null });
+    } else {
+      this.setState({ selectedTopicId: Number(event.target.id) });
+    }
+  }
+
+  render() {
+    const currentId = this.state.selectedTopicId;
+    const listTopics = this.topics.map(topic =>
+      <div key={topic.id} onClick={this.handleClick}>
+        <li id={topic.id} >
```

Comment:
This is fine, but instead of using an `id` attribute on the element you can directly pass the `id` to the `handleClick` method.

```JSX
<div key={topic.id} onClick={() => this.handleClick(topic.id)}>
      <li>
```

Reply:
Good advice, thanks!